### PR TITLE
add Home page

### PR DESCRIPTION
**Pull request by NVyrsta**

### CODE REVIEW

File: src/components/Header/Header.scss [hunk 1]
```diff
@@ -1,29 +1,118 @@
...
+    gap: 64px;
+  }
+}
+
 .header {
-  box-sizing: border-box;
-  margin: 0 auto;
+  font-family: 'Mont', sans-serif;
   max-width: 1440px;
-  border-bottom: 1px solid #ddd;
```
Comment (OlehSharha) [inline src/components/Header/Header.scss:5]: make ur header width 100%, not  fixed

File: src/components/ShopByCategory/ShopByCategory.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import Grid from "@mui/material/Grid";
+import './ShopByCategory.scss';
+import { CategoryCard } from '../CategoryCard';
+
+import categoryPhonesImg from "../../img/categories/category-phones.jpg";
+import categoryTabletsImg from "../../img/categories/category-tablets.jpg";
+import categoryAccessoriesImg from "../../img/categories/category-accessories.jpg";
+
+export const ShopByCategory: React.FC = () => {
+  const phonesAmount = 95;
```
Comment (OlehSharha) [inline src/components/ShopByCategory/ShopByCategory.tsx:11]: i dont know better this or not, not use hardcoded value, but get array length from server and display it(

File: src/modules/HomePage/HomePage.tsx [hunk 3]
```diff
@@ -1,20 +1,36 @@
...
+      <Grid item tablet={12} tabletXL={12} desktop={12} mobile={12}>
+        <ShopByCategory />
+      </Grid>
+
+      <Grid item tablet={12} tabletXL={12} desktop={12} mobile={12}>
+        <HotPrices />
+      </Grid>
+
+      {/* <Grid item tablet={6} tabletXL={4} desktop={3} mobile={12}>
         <p className="test">Item</p>
```
Comment (OlehSharha) [inline src/modules/HomePage/HomePage.tsx:34]: delete unused comments :)

File: src/modules/PhonesPage/PhonesPage.tsx [hunk 4]
```diff
@@ -28,7 +28,8 @@ export const PhonesPage = () => {
   const getPhonesFromServer = async (searchParam) => {
     setIsLoading(true);
     try {
-      const { amount, phones} = await getPhonesData(searchParam);
+      const { amount, phones } = await getPhonesData(searchParam);
+      console.log(searchParam, "searchParam", amount, phones);
```
Comment (OlehSharha) [inline src/modules/PhonesPage/PhonesPage.tsx:32]: if u don`t need this, delete too)

**Review by OlehSharha — APPROVED** [review]
Just little things)

File: src/components/Header/Header.scss (hunk 1 above)
Comment (NVyrsta) [reply src/components/Header/Header.scss:5]: I don't get it

**Pull request merged**